test-smoke/electron/test/renderer.ts

```diff
@@ -60,11 +60,19 @@ console.log(webFrame.getZoomLevel());
 webFrame.setVisualZoomLevelLimits(50, 200);
 webFrame.setLayoutZoomLevelLimits(50, 200);
 
-webFrame.setSpellCheckProvider('en-US', true, {
-	spellCheck: text => {
-		return !(require('spellchecker').isMisspelled(text));
-	}
-});
+webFrame.setSpellCheckProvider('en-US', {
+  spellCheck (words, callback) {
+    setTimeout(() => {
+      let misspeltWords = []
```

Comment:
* nit: `misspelt' is valid but has 1/28th the Google hits as 'misspelled', _and_ the spellchecker API uses the word 'misspelled', so follow the Principle of Least Surprise and use it here too
 * `const` in JS means "we won't change the address this pointer points to" rather than "this variable's value is immutable", so this can & should be `const` rather than `let`

Reply:
I was actually using misspelt in the updated spellcheck API PR in electron. Changed that also.